README.rst

```diff
@@ -29,6 +29,20 @@ Dependencies
 * gcc/gfortran or Intel compilers (clang might not work out of the box), if compiling from source
 * ducc0_, healpy, Cython, astropy, numpy, scipy, matplotlib, pyyaml, h5py, Pillow (Python Image Library)
 
+On MacOS, and other systems with non-traditional environments, you should specify the following standard environment variables:
+
+* ``CC``: C compiler (example: ``gcc``)
+* ``CXX``: C++ compiler (example: ``g++``)
+* ``FC``: Fortran compiler (example: ``gfortran``)
+
+We recommend using ``gcc`` installed from Homebrew to access these compilers.
```

Comment:
Could you add a warning that the user should specify the full path to GNU gcc? Otherwise, "gcc" will point to clang.